rotkehlchen/inquirer.py

```diff
@@ -388,8 +440,84 @@ def find_usd_price(
             Inquirer._cached_current_price[cache_key] = CachedPriceEntry(price=price, time=ts_now())  # noqa: E501
             return price
 
+        if is_known_protocol or underlying_tokens is not None:
+            usd_price = get_underlying_asset_price(token)  # type: ignore
+            if usd_price is None:
+                price = Price(ZERO)
+            else:
+                price = Price(usd_price)
+            Inquirer._cached_current_price[cache_key] = CachedPriceEntry(
+                price=price,
+                time=ts_now(),
+            )
+            return price
+
         return instance._query_oracle_instances(from_asset=asset, to_asset=A_USD)
 
+    def find_uniswap_v2_lp_price(
+        self,
+        token: Union[EthereumToken, UnknownEthereumToken],
+    ) -> Optional[Price]:
+        """
+        First thing should be query oracles for prices. It it files then we
+        calculate price for a LP token. This one is
+        value = (Total value of liquidity pool)/(Current suply of LP tokens)
+        We need:
+        - Price of token 0
+        - Price of token 1
+        - Pooled amount of token 0
+        - Pooled amount of token 1
+        - Total suply of of pool token
+        """
+        address = token.ethereum_address
+        contract = EthereumContract(address=address, abi=UNISWAP_V2_LP_ABI, deployed_block=0)
+        methods = ['token0', 'token1', 'totalSupply', 'getReserves', 'decimals']
+
+        if self._ethereum is None:
+            log.debug(
+                'Tried to query uniswap v2 LP position price but '
+                'Inquierer is not correctly initializen',
+            )
+            return None
+
+        try:
+            output = multicall_2(
+                ethereum=self._ethereum,
+                require_success=True,
+                calls=[(address, contract.encode(method_name=method)) for method in methods],
+            )
+        except RemoteError:
+            log.debug('Remote error calling multicall contract for uniswap v2 lp token properties')
```

Comment:
this is an error. Also include more information. Such as the address
```suggestion
            log.error(f'Remote error calling multicall contract for uniswap v2 lp token {token.address} properties')
```